packages/shared/providers.yaml

```diff
@@ -702,7 +702,7 @@ pipedrive:
     authorization_url: https://oauth.pipedrive.com/oauth/authorize
     token_url: https://oauth.pipedrive.com/oauth/token
     proxy:
-        base_url: https://${connectionConfig.subdomain}.pipedrive.com
+        base_url: https://api.pipedrive.com
```

Comment:
@joepvdbogaert curious here: the api subdomain is always static for Pipedrive?

Reply:
I believe so, certainly was the case for my account. Cannot seem to find anything about this explicitly in the docs, but their examples also just use this domain. Do you have a (test) account there yourself to validate?

Reply:
Mm I guess they do advise to use a subdomain in the request: https://pipedrive.readme.io/docs/core-api-concepts-requests#url-naming. Will close this and take another look later what's happening.